src/lib/authserver/ui/new_krb_inst.rb

```diff
@@ -153,28 +153,28 @@ def ok_handler
     out, ok = ldap.create_person(kdc_dn_prefix, 'Kerberos KDC Connection', dir_suffix)
     MITKerberos.append_to_log('%s' % out)
     if !ok
-      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST2/y2log'))
       UI.ReplaceWidget(Id(:busy), Empty())
       return
     end
     out, ok = ldap.change_password(kdc_dn,kdc_pass)
     MITKerberos.append_to_log('%s' % out)
     if !ok
-      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST2/y2log'))
```

Comment:
Please, move the path the translation.

```suggestion
      Popup.Error(format(_('Failed to create Kerberos KDC connection user! Log output may be found in %{y2log}'), y2log: y2log))
```